testdata/baselines/reference/submoduleAccepted/conformance/instantiateTemplateTagTypeParameterOnVariableStatement.errors.txt.diff

```diff
@@ -0,0 +1,29 @@
+--- old.instantiateTemplateTagTypeParameterOnVariableStatement.errors.txt
++++ new.instantiateTemplateTagTypeParameterOnVariableStatement.errors.txt
+@@= skipped -0, +0 lines =@@
+-<no content>
++instantiateTemplateTagTypeParameterOnVariableStatement.js(12,5): error TS2322: Type 'T' is not assignable to type 'string'.
++instantiateTemplateTagTypeParameterOnVariableStatement.js(12,24): error TS2345: Argument of type 'string' is not assignable to parameter of type 'T'.
++  'T' could be instantiated with an arbitrary type which could be unrelated to 'string'.
++
++
++==== instantiateTemplateTagTypeParameterOnVariableStatement.js (2 errors) ====
++    /**
++     * @template T
++     * @param {T} a
++     * @returns {(b: T) => T}
++     */
++    const seq = a => b => b;
++    
++    const text1 = "hello";
++    const text2 = "world";
++    
++    /** @type {string} */
++    var text3 = seq(text1)(text2);
++        ~~~~~
++!!! error TS2322: Type 'T' is not assignable to type 'string'.
```

Comment:
`@template` still needs a fix to type instantation -- I believe it's inspecting parents at some point and failing.